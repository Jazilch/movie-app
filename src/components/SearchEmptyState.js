'use es6';

import React from 'react';
import styled from 'styled-components/macro';

const SearchContentWrapper = styled.div`
  text-align: center;
`;

const SearchEmptyState = () => {
  return (
    <SearchContentWrapper>
      <h3>No results</h3>
      <p>Try refining your search results</p>
    </SearchContentWrapper>
  );
};

export default SearchEmptyState;
