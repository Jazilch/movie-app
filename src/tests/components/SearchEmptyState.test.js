/* eslint-disable no-undef */

'use es6';

import React from 'react';
import { shallow } from 'enzyme';
import SearchEmptyState from '../../components/SearchEmptyState';

describe('SearchEmptyState component', () => {
  it('correctly renders SearchEmptyState', () => {
    const searchemptystate = shallow(<SearchEmptyState />);
    expect(searchemptystate).toMatchSnapshot();
  });
});
