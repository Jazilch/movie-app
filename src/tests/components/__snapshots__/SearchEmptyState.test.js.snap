// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchEmptyState component correctly renders SearchEmptyState 1`] = `ShallowWrapper {}`;
